modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -0,0 +1,476 @@
+* The Avro duration logical type is ignored.
+* The Avro null type is ignored and not represented in the Iceberg schema.
+* Recursive types are not supported.
+--
+
+Protobuf::
++
+--
+|===
+| Protobuf type | Iceberg type
+
+| bool | boolean
+| double | double
+| float | float
+| int32 | int
+| sint32 | int
+| int64 | long
+| sint64 | long
+| sfixed32 | int
+| sfixed64 | int
+| string | string
+| bytes | binary
+| map | map
+|===
+
+* Repeated values are translated into Iceberg `array` types.
+* Enums are translated into Iceberg `int` types based on the integer value of the enumerated type.
+* `uint32` and `fixed32` are translated into Iceberg `long` types as that is the existing semantic for unsigned 32-bit values in Iceberg.
+* `uint64` and `fixed64` values are translated into their Base-10 string representation.
+* The `timestamp` type in Protobuf is translated into `timestamp` in Iceberg.
+* Messages are converted into Iceberg structs.
+* Recursive types are not supported.
+--
+======
+
+=== Schema evolution
+
+Redpanda supports schema evolution for Avro and Protobuf schemas in accordance with the https://iceberg.apache.org/spec/#schema-evolution[Iceberg specification^]. Permitted schema evolutions include reordering fields and promoting field types. When you update the schema in Schema Registry, Redpanda automatically updates the Iceberg table schema to match the new schema.
+
+For example, if you produce records to a topic `demo-topic` with the following Avro schema:
+
+.schema_1.avsc
+[,avro]
+----
+{
+  "type": "record",
+  "name": "ClickEvent",
+  "fields": [
+    {
+      "name": "user_id",
+      "type": "int"
+    },
+    {
+      "name": "event_type",
+      "type": "string"
+    }
+  ]
+}
+----
+
+[,bash]
+----
+rpk registry schema create demo-topic-value --schema schema_1.avsc
+
+echo '{"user_id":23, "event_type":"BUTTON_CLICK"}' | rpk topic produce demo-topic --format='%v\n' --schema-id=topic
+----
+
+Then, you update the schema to add a new field `ts`, and produce records with the updated schema:
+
+.schema_2.avsc
+[,avro]
+----
+{
+  "type": "record",
+  "name": "ClickEvent",
+  "fields": [
+    {
+      "name": "user_id",
+      "type": "int"
+    },
+    {
+      "name": "event_type",
+      "type": "string"
+    }.
+    {
+      "name": "ts",
+      "type": [
+          "null",  
+          { "type": "string", "logicalType": "date" }
+        ],
+      "default": null  # Default value for the new field
+    }
+  ]
+}
+----
+The `ts` field can be either null or a string representing a date. The default value is null.
+
+[,bash]
+----
+rpk registry schema create demo-topic-value --schema schema_2.avsc
+
+echo '{"user_id":858, "event_type":"BUTTON_CLICK", "ts":{"string":"2025-02-26T20:05:23.230ZZ"}}' | rpk topic produce demo-topic --format='%v\n' --schema-id=topic
+----
+
+xref:manage:iceberg/query-iceberg-topics.adoc[Querying the Iceberg table] for `demo-topic` includes the new column `ts`:
```

Comment:
I think the link is kind of distracting here.

Reply:
Updated